### ISSUE TITLE
[REQUEST] Implement missing `none` DRC preset with binary edit.

### ISSUE THREAD

**Issue by pcroland** [pcroland/deew#29]

**Comment by pcroland** [comment]
https://forum.doom9.org/showpost.php?p=1973074&postcount=144

**Comment by ValZapod** [comment]
For TrueHD there is an ON flag and a ability to write -31. That is none, right?

> F.W.I.W: adding the line "<custom_dialnorm>-31</custom_dialnorm>" to the XML configuration file apparently avoids the undesired dialnorm thing.
> Still, no way to set the DRC value to "none".

**Comment by pcroland** [comment]
You can use -31 for DD, DDP and TrueHD aswell. The on flag is only for TrueHD 2.0, don't know why. Dolby doesn't plan to add the missing `none` preset.

**Comment by ValZapod** [comment]
>Dolby doesn't plan to add the missing none preset.

That is just -31, right? They are not planning to add it cause that should be turned off on the AVR.

TrueHD 2.0?? You mean DVD-A MLP was 1.0? Or you are talking about thd 1.0 from here? https://trac.ffmpeg.org/ticket/667

**Comment by pcroland** [comment]
dialnorm -31 is not the same as a disabled drc.

**Comment by ValZapod** [comment]
What is the difference? Again, ffmpeg does not support encoding or decoding THD taking DRC into account.

**Comment by ValZapod** [comment]
>The on flag is only for TrueHD 2.0

That is false, not stereo works too. 2.0 is a downmix of 5.1, the option is not for it.

See, this is left "DRC default on" toggle on, and right off.

![image](https://user-images.githubusercontent.com/31514790/216791882-302b8a9d-33bd-4ae2-bc64-fb3c99bc4a4a.png)

![image](https://user-images.githubusercontent.com/31514790/216791952-3048d7e0-80f0-4ecb-815f-c3151ae76a86.png)
